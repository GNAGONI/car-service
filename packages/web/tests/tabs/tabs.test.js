import React from 'react'
import renderer from 'react-test-renderer'
import Tabs from '../../components/tabs/tabs'

it('renders correctly', () => {
  const tree = renderer.create(<Tabs />).toJSON()
  expect(tree).toMatchSnapshot()
})
