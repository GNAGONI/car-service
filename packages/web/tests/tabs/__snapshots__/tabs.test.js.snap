// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <ul
    className="tabset"
  />
  <div
    className="tab-content"
  >
    <div
      className="tab "
    />
  </div>
</div>
`;
